tests/helpers/perf_run_helper.rb

```diff
@@ -238,22 +248,14 @@ def puppet_infrastructure_tune_current
   #   restart_puppetserver
   #
   def restart_puppetserver
-    if ENV["PUPPET_GATLING_SCALE_RESTART_PUPPETSERVER"].eql?("false")
-      puts "PUPPET_GATLING_SCALE_RESTART_PUPPETSERVER is set to false..."
-      puts "Skipping pe-puppetserver service restart..."
-      puts
-    else
-      puts "Restarting pe-puppetserver service..."
-      puts
-
-      on master, "service pe-puppetserver restart"
-
-      puts "Sleeping..."
-      puts
+    puts "Restarting pe-puppetserver service..."
+    puts
+    on master, "service pe-puppetserver restart"
 
-      sleep 60
-    end
+    puts "Sleeping..."
+    puts
 
+    sleep 60
```

Comment:
Should this entire method be removed since the `restart_puppet_server` command from master-manipulator is now being used?

Reply:
Yes, I had actually deleted it, then realized I wanted the messages so did an undo, grabbed the messages, then forgot to re-delete it :)